src/services/BadgesService.ts

```diff
@@ -310,4 +313,36 @@ export class BadgesService {
       throw new Error(msg)
     }
   }
+
+  static toGovernanceBadge(otterspaceBadge: OtterspaceBadge) {
+    const { name, description, image } = otterspaceBadge.spec.metadata
+    const badge: Badge = {
+      name,
+      description,
+      createdAt: otterspaceBadge.createdAt,
+      owner: otterspaceBadge.owner?.id,
+      image: BadgesService.getGithubImagePermalink(name, image),
+      status: toBadgeStatus(otterspaceBadge.status),
+      statusReason: toBadgeStatusReason(otterspaceBadge.statusReason),
+      isPastBadge: isPastBadge(otterspaceBadge),
+      transactionHash: otterspaceBadge.transactionHash || '',
+    }
+    return badge
+  }
+
+  static toGovernanceBadgeSpec(otterspaceBadges: OtterspaceBadge[]): GovernanceBadgeSpec {
+    if (otterspaceBadges.length === 0) throw new Error('No badges found')
+    const { name, description, image } = otterspaceBadges[0].spec.metadata
+    const badges = otterspaceBadges.map(BadgesService.toGovernanceBadge)
+    return { name, description, image: BadgesService.getGithubImagePermalink(name, image), badges }
+  }
+
+  static getGithubImagePermalink(badgeName: string, ipfsLink: string): string {
```

Comment:
This method doesn't need to be static right? It is only used here so it can be private. And you can call it `this.getGithubImagePermalink`?